drivers/windows/R/resource.R

```diff
@@ -13,9 +13,9 @@ cluster_info_wpia_hn <- function() {
   resources <- list(
     max_cores = 32,
     max_ram = 512,
-    queues = c("AllNodes", "Training"),
+    queues = c("AllNodes", "FastQueue"),
```

Comment:
I wonder if we should use `Testing` and not `FastQueue` here, to emphasise that the queue is for testing work?

Reply:
Yeah, good call - perhaps "FastQueue" has other surface-level implications too...